### ISSUE TITLE
Add bundle outputs to claim

### ISSUE THREAD

**Issue by vdice** [deislabs/porter#457]
https://github.com/deislabs/cnab-spec/pull/217 added an outputs section to the claim spec.

This ticket is for Porter to add the corresponding outputs section to the the claims it generates after each action.

**Comment by vdice** [comment]
Depends on https://github.com/deislabs/cnab-go/pull/66 (and updating our cnab-go dep once merged/released)

**Comment by carolynvs** [comment]
What we have been doing is to not let waiting for a cnab-go change to hold up the Porter work. So when you need a change in cnab-go, submit the PR, then immediately do the work you need in Porter relying on your PR's branch and update Gopkg.toml to use your fork. Then once the cnab-go PR is merged, remove the alternate source/branch. If we need multiple PRs merged into one, we can discuss and work on a coordinated strategy.

Otherwise this seriously slows down our work on Porter, and also prevents us from adequately verifying our PRs to cnab-go. Often we don't realize that we need more in those PR until we use them in Porter. 😀
